src/js/player.js

```diff
@@ -568,6 +569,9 @@ class Player extends Component {
       el.appendChild(tag);
 
       playerElIngest = this.playerElIngest_ = el;
+      Object.keys(el).forEach((k) => {
+        tag[k] = el[k];
```

Comment:
Not sure if we want this or not, but it seems like we did. This brings properties that are set in js from the video-js object over to the video object. In my case I was using `mediaEl.src = ...;` (`mediaEl` being the `video-js` element) in js. Then initializing the player. This would cause the `src` property to be lost which I do not think is correct.